### PR TITLE
docs: Fix formatting mistake

### PR DESCRIPTION
**Pull request by xordspar0**
Part of this section was a code block, and part of it was absorbed into the preceding normal paragraph.

Before/after:

image

### CODE REVIEW

File: docs/source/markdown/podman-create.1.md [hunk 1]
```diff
@@ -1121,13 +1121,11 @@ required for VPN, without it containers need to be run with the --network=host f
 Environment variables within containers can be set using multiple different options:  This section describes the precedence.
 
 Precedence Order:
```
Comment (mheon) [inline docs/source/markdown/podman-create.1.md:1123]: Suggestion: maybe we should add "from least to greatest" here to indicate what the actual ordering is.

**Comment by mheon** [comment]
Can you make the same change to `podman-run.1.md` in the same directory?

**Comment by rhatdan** [comment]
You should also mention 
--http-proxy=true|false
Which will set some environment variables as well.

**Comment by rhatdan** [comment]
We really should have a test that makes sure the hierarchy works, and stays working.

File: docs/source/markdown/podman-create.1.md (hunk 1 above)
Comment (xordspar0) [reply docs/source/markdown/podman-create.1.md:1123]: Great idea.

**Comment by xordspar0** [comment]
I've made most of the changes. I'm looking into how --http-proxy=true|false fits in now.

**Comment by mheon** [comment]
LGTM once the HTTP proxy change makes it in.

**Comment by xordspar0** [comment]
Done. Please take a look at what I did for the --http-proxy docs. I can squash these if needed, but I think it makes sense to keep them separate.

**Comment by xordspar0** [comment]
For anyone curious, the --http-proxy and the --host-envs code are in the same function: https://github.com/containers/podman/blob/be5219ab9529c596421dd19bb61cdd2ced267ece/vendor/github.com/containers/common/pkg/config/config.go#L706-L722

**Comment by rhatdan** [comment]
Thanks @xordspar0 for the PR.

**Pull request merged**